### ISSUE TITLE
"403 No Permission"

### ISSUE THREAD

**Issue by MouloudP** [everyday-as/gmodstore-issues#1179]
### Expected Behavior

I wanted make a "site ticket"

### Actual Behavior

I get a "403 No Permission" when i want make a ticket, edit an addon / media / tabs / collapsibles / support settings
And also i don't have the "version tab"
![image](https://user-images.githubusercontent.com/44133428/163283311-8cb01f57-8249-45f9-8a69-f240fa924a52.png)
![image](https://user-images.githubusercontent.com/44133428/163283591-3e5115bf-485c-44b5-91e9-c7509812f54b.png)

### Reproduction Steps

```markdown
- https://www.gmodstore.com/help/tickets/create/site
or
- https://www.gmodstore.com/dashboard/addons/aden-character-system/edit
```

### Browsers

Google chrome

### Example URL

https://www.gmodstore.com/help/tickets/create/site

### GmodStore Version

v11.0.1

**Comment by Marcuzz** [comment]
Thanks,

This is fixed in 11.0.2, which is being released shortly.

**Comment by Marcuzz** [comment]
11.0.2 is out now, I would appreciate it if you could verify that it's fixed 😄

**Comment by MouloudP** [comment]
I don't have the permission issue ! Yes but i have an other problem now x)

**Comment by MouloudP** [comment]
![image](https://user-images.githubusercontent.com/44133428/163285168-51605a23-1ca6-4f8a-a6f2-724ffd457a4e.png)

**Comment by MouloudP** [comment]
71e315a9588044d88e35ec7dbaf9d821
I wanted update a tab

**Comment by Marcuzz** [comment]
Thanks. I'll look into it.

**Comment by Marcuzz** [comment]
My colleague @J0sh0nat0r will look into this once he's back, he has more knowledge about this particular system than me 😄

**Comment by MouloudP** [comment]
Okay thank you ! Thanks for your answer ! (And for your work 😃)

**Comment by J0sh0nat0r** [comment]
This should be fully resolved in 11.0.3! Feel free to re-open if you're still encountering issues :)
